### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Logic/TransportStream/RegionCompositionSegment.cs [hunk 1]
```diff
@@ -49,5 +49,4 @@ public RegionCompositionSegment(byte[] buffer, int index, int regionLength)
             }
         }
     }
-
```
Comment (ivandrofly) [inline src/Logic/TransportStream/RegionCompositionSegment.cs]: Outside class

Comment (niksedk) [inline src/Logic/TransportStream/RegionCompositionSegment.cs]: What do you mean by "Outside class" ?

Comment (ivandrofly) [inline src/Logic/TransportStream/RegionCompositionSegment.cs]: You said  before tab after method close is useful when adding new method

Comment (niksedk) [inline src/Logic/TransportStream/RegionCompositionSegment.cs]: Ah, I thought it was something like "inner class" I did not know about ;)

Comment (ivandrofly) [inline src/Logic/TransportStream/RegionCompositionSegment.cs]: > Ah, I thought it was something like "inner class" I did not know about ;)

:smile:

**Pull request merged**